config/nginx/nginx.conf

```diff
@@ -0,0 +1,19 @@
+http {
+
+    server {
+
+        listen 9003;
```

Comment:
why port `9003` here? can we use instead the standard `9100` as defined by prometheus (https://github.com/prometheus/prometheus/wiki/Default-port-allocations), or is this to avoid `:9100` conflicts with other services?